tests/tasks/teardown/kitctl.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: teardown
+  namespace: tekton-pipelines
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.17.0"
+    tekton.dev/categories: Kubernetes
+    tekton.dev/tags: CLI, kubectl
+    tekton.dev/displayName: "kubernetes actions"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: |
+    Tear down the guest cluster.
+  params:
+  - name: name
+    default: "guest"
+    description: Name of the guess cluster
+  steps:
+  - name: teardown
+    image: bitnami/kubectl
```

Comment:
https://hub.docker.com/r/alpine/k8s

Reply:
This bitnami image is 25% smaller than the k8s toolbox image. Shall we replace the image later if we need tools other than `kubectl`?